scripts/helpers/deploy.js

```diff
@@ -193,91 +197,108 @@ async function deployWithoutProxy(nameInState, artifactName, deployer, construct
   const netId = await web3.eth.net.getId()
   const state = readNetworkState(network.name, netId)
 
-  console.log(`Deploying ${artifactName}...`)
-  if (constructorArgs.length > 0) {
-    console.log(`Constructor args: ${constructorArgs}`)
-  }
+  process.stdout.write(`Deploying ${artifactName} (NO proxy)... `)
 
   const contract = await deployContract(artifactName, constructorArgs, deployer)
-  console.log(`${artifactName} (NO proxy): ${contract.address}`)
 
   const gasUsed = await getDeploymentGasUsed(contract)
-  console.log(`    gas used ${gasUsed}`)
+  console.log(`done: ${contract.address} (gas used ${gasUsed})`)
```

Comment:
:brain:  I'd suggest to have the logs not only printed but dumped to the file, not a strong opinion tho